containers/podman-desktop
# chore(tests): run e2e podman machine tests only under condition

If `TEST_PODMAN_MACHINE=true` 

### What does this PR do?
Unifies the condition for all podman machine e2e tests. 
### Screenshot / video of UI



### What issues does this PR fix or reference?
#9149 


### How to test this PR?
On windows, set `TEST_PODMAN_MACHINE=true` and run the tests, all podman machine tests should be run, if not set, they should not be run.
